assets/batch_score/components/driver/src/batch_score_oss/common/scoring/scoring_client_factory.py

```diff
@@ -50,28 +42,24 @@ def setup_scoring_client(
         tally_handler = _setup_tally_handler(configuration)
         endpoint_type = configuration.get_endpoint_type()
 
-        if endpoint_type in [EndpointType.AOAI, EndpointType.Serverless]:
+        if endpoint_type in [EndpointType.Serverless]:
             get_logger().info("Using LLM scoring client.")
             return _setup_aoai_scoring_client(
                 configuration=configuration,
                 header_provider=header_provider,
                 tally_handler=tally_handler)
 
-        if endpoint_type == EndpointType.BatchPool:
-            get_logger().info("Using batch pool scoring client.")
-            return _setup_pool_scoring_client(
-                configuration=configuration,
-                header_provider=header_provider,
-                tally_handler=tally_handler,
-                token_provider=token_provider,
-                routing_client=routing_client)
-
         get_logger().info("Using MIR scoring client.")
         return _setup_mir_scoring_client(
             configuration=configuration,
             header_provider=header_provider,
             tally_handler=tally_handler)
 
+        # get_logger().info("Using null endpoint scoring client.")
```

Comment:
you might want to enable null endpoint setup here if you'd like to use them.

Reply:
Done